### PR TITLE
Adds a "tip of the round" system

### PR DESCRIPTION
**Pull request by tkdrg**
These are displayed 30 seconds before roundstart.
They're randomly picked from config/tips.txt (currently a placeholder).
If you can contribute with more tips, please do.

### CODE REVIEW

**Comment by SuperSayu** [comment]
Alt-click on an adjacent square to see its contents in the top right pane.
While observing or as a ghost, double click on people, bots, or the singularity to follow them.
Ghosts can click on active teleporters, portals, or the gateway to jump to its destination!
Ghosts can double-click their corpse, or the object containing their corpse (such as the cloning scanner), to re-enter it.
Ghosts can see inside storage items that have been left on the ground.

**Comment by paprka** [comment]
Adjacent *tile not square

File: config/tips.txt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+You can drag yourself onto tables to climb on them. This takes a moment and requires two free hands.
+You can drag other players onto yourself to open the strip menu, letting you remove their equipment or force them to equip something.
+Aliens are by far the most powerful close-quarters combat menace. Fight them at a distance to gain the upper hand!
+Aliens take double damage from all burn sources, such as lasers and fires. Flamethrowers are extremely effective against them.
+As an alien, your most powerful weapon is the facehugger. It can instantly win any fight, but can be completely blocked by certain headgear. Strip downed victims before applying one.
+Monkeys can still wear some human items, such as masks and backpacks, but they cannot bring anything with them when vent crawling.
+Nuclear operatives are very effective in a blitzkrieg tactic: attacking almost as soon as the round starts and well before the crew is prepared.
+Lizard men, slime men, and monkeys all lose protection from the AI's laws as soon as they cause human harm.
```
Comment (Tenebrosity) [inline config/tips.txt]: isn't this /tg/ specific

File: config/tips.txt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Many monsters won't attack you if you're unconscious, but a medibot will still heal you. Keep one around when you're mining and never worry about dying again.
+Two people are critical and need your help, but you don't have time for two trips? Pull one and use the grab intent to grab the other. This will make you slower, but you will be able to take them both.
+When using the grab intent, grab someone by clicking on them then upgrade your grab by clicking on your hand slot. Holding them by the hands lets you throw them or table them.
+Can't seem to get a patient into the cryo tube or sleepers? Just drag their sprite to it to put them in it instantly. No more needing to shuffle around pulling them into it awkwardly.
+EMP blasts have a chance to make machines explode!
+Targeting your opponent's mouth in a fistfight will give you a greater chance of knocking them unconscious.
+As a security officer, examining a crew member with a secHUD allows you to set their status to arrest. You can even give a reason for the arrest warrant, as well.
+Clowns are extremely clumsy and will often misfire weapons.
+Hulk's can't fire weapons because of their meaty fingers.
```
Comment (Tenebrosity) [inline config/tips.txt]: >Hulk's

File: config/tips.txt [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+The Librarian system is much more robust than most people think! Mess around with it, see if you can log who takes out the books for radio-reading.
+Use a pair of handcuffs on a pair of orange shoes (standard prisoner issue) to chain them together.
+Revolutionaries brainwashed by Rev leaders can have their opinions swayed back to nanotrasen loyalty by enough heavy objects to the head.
+Husked corpses that have been drained of their genomes by a changeling also lack blood in their veins, unlike burned or space frozen corpses.
+The stethoscope's utility extends beyond lung and heart checkups. They can also be used to crack certain safes.
+Chloral Hydrate can be counteracted by having coffee in your system!
+The AI system integrity restorer in both the RD's office and on the bridge can revive a dead AI loaded onto an intellicard.
+If an anomaly appears, scan it with an analyzer and then ping the frequency it gives you with a remote signaling device. 
+Various creatures can crawl through air vents by holding alt while left clicking them.
+Wizards are never friendly and should not be seen as friendly. Expect chaos to ensue shortly after they "declare" themselves as such.
```
Comment (Tenebrosity) [inline config/tips.txt]: This is misleading and opinionated

**Comment by tkdrg** [comment]
Conflicts de-conflicted.

**Comment by RemieRichards** [comment]
@tkdrg Ready to go?

**Comment by tkdrg** [comment]
@RemieRichards Should be

**Pull request merged**